CONTRIBUTING.adoc

```diff
@@ -26,13 +36,13 @@ Once you've completed the web form, simply add the following in a comment on you
     I have signed and agree to the terms of the SpringSource Individual
     Contributor License Agreement.
```

Comment:
Looks like we can keep all the words on a single line.
Any objections do not do that on merge?

Yes, minor, of course, but at a glance I would use this sentence in my PR as is: with that extra new line.